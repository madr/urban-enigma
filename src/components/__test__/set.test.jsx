import React from "react";
import renderer from "react-test-renderer";
import Set from "../set";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Set
        exercise={"Press"}
        sets={[
          { id: 0, reps: 5, weight: 60, rating: 4 },
          { id: 1, reps: 5, weight: 60, rating: 5 },
          { id: 2, reps: 5, weight: 60, rating: 3 }
        ]}
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
