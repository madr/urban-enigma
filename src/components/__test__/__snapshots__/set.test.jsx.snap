// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <tr>
    <th>
      Press
    </th>
    <th>
      70
       kg 
      <small>
        1rm
      </small>
    </th>
  </tr>,
  <tr
    className=""
  >
    <td>
      <big>
        5
      </big>
       reps
    </td>
    <td>
      <big>
        60
      </big>
       kg
    </td>
  </tr>,
  <tr
    className=""
  >
    <td>
      <big>
        5
      </big>
       reps
    </td>
    <td>
      <big>
        60
      </big>
       kg
    </td>
  </tr>,
  <tr
    className=""
  >
    <td>
      <big>
        5
      </big>
       reps
    </td>
    <td>
      <big>
        60
      </big>
       kg
    </td>
  </tr>,
]
`;
